rest/utilities_testing.go

```diff
@@ -139,6 +139,33 @@ func (rt *RestTester) GetDatabase() *db.DatabaseContext {
 	return nil
 }
 
+func (rt *RestTester) MustWaitForDoc(docid string) {
+	err := rt.WaitForDoc(docid)
+	if err != nil {
+		panic(fmt.Sprintf("Error waiting for doc: %v", docid))
```

Comment:
A panic is going to terminate the entire test run on failure, isn't it?  If you're trying to avoid having to check for error in every caller, could consider taking testing.T as a parameter and assert?

Reply:
Ok, will make that change